README.md

```diff
@@ -9,9 +9,17 @@ Any open bugs and/or feature requests are tracked in [GitHub issues](../../issue
 
 Interested in contributing to this role, please see [CONTRIBUTING](CONTRIBUTING.md)
 
+## Documentation
+
+* User guide: [[Parser Directives]](docs/directives/parser_directives.md)
+* User guide: [[Template Directives]](docs/directives/template_directives.md)
```

Comment:
Should not be part of the this release.. this is coming in a subsequent release

Reply:
Removed